i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md

```diff
@@ -2,74 +2,856 @@
+| --filter-type            | Filter backups by type (regexp can be used).                                                                                                                                                                                             | Mode |
+| --unknown-status         | Set unknown threshold for status. You can use the following variables: %{has\_backup}, %{last\_error}, %{exit\_state}, %{type}                                                                                                           | Mode |
+| --warning-status         | Set warning threshold for status. You can use the following variables: %{has\_backup}, %{last\_error}, %{exit\_state}, %{type}                                                                                                           | Mode |
+| --critical-status        | Set critical threshold for status (Default: '%{has\_backup} eq "yes" and %{exit\_state} ne "SUCCESS" and %{last\_error} ne "NO\_ERROR"'). You can use the following variables: %{has\_backup}, %{last\_error}, %{exit\_state}, %{type}   | Mode |
+| --warning-* --critical-* | Thresholds. Can be: 'time-last-execution'.                                                                                                                                                                                               | Mode |
+
+</TabItem>
+<TabItem value="Connection-Time" label="Connection-Time">
+
+| Option     | Description                            | Type |
+|:-----------|:---------------------------------------|:-----|
+| --warning  | Threshold warning in milliseconds.     | Mode |
+| --critical | Threshold critical in milliseconds.    | Mode |
+
+</TabItem>
+<TabItem value="Connections-Number" label="Connections-Number">
 
-All prerequisites needed to run mysql plugins are packaged and installed with
-Centreon Entreprise Server: perl-DBI , perl-DBD-mysql , mysql.
+| Option                   | Description                                                                     | Type |
+|:-------------------------|:--------------------------------------------------------------------------------|:-----|
+| --add-databases          | Add threads by databases.                                                       | Mode |
+| --warning-* --critical-* | Thresholds. Can be: 'usage', 'usage-prct' (%), 'database-threads-connected'.    | Mode |
 
-## Centreon Configuration
+</TabItem>
+<TabItem value="Database-Size" label="Database-Size">
 
-### Create a new MariaDB server
+| Option                   | Description                                                                                                            | Type |
+|:-------------------------|:-----------------------------------------------------------------------------------------------------------------------|:-----|
+| --filter-database        | Filter database to checks (Can be a regexp).                                                                           | Mode |
```

Comment:
?

Reply:
Some remaining help messages to improve :/